Dockerfile

```diff
@@ -47,4 +47,6 @@ RUN useradd -m stringer
 RUN chown -R stringer:stringer /app
 USER stringer
 
+ENV RAILS_SERVE_STATIC_FILES=true
```

Comment:
@guidopetri when do we decide to put an env variable in the `Dockerfile` vs the `.docker-compose.yml`?

Reply:
Generally any "default" things that don't need customization, or that customization significantly alters the inner workings of the app, should go in the Dockerfile imo. The env var can still be overwritten in the docker compose file though. So e.g. we might set the default port to 8080 in the Dockerfile, but then also overwrite it to 80 in the compose